compiler/env/CPU.hpp

```diff
@@ -26,11 +26,12 @@
 
 namespace TR
 {
-class CPU : public OMR::CPUConnector
+class OMR_EXTENSIBLE CPU : public OMR::CPUConnector
    {
    public:
 
    CPU() : OMR::CPUConnector() {}
+   CPU(const OMRProcessorDesc& processorDescription) : OMR::CPUConnector(processorDescription) {}
```

Comment:
These both will require Doxygen documentation since we expect them to be part of the public compiler API.